proto/backend.proto

```diff
@@ -134,6 +134,64 @@ message DataResponse {
   bytes jsonMeta = 3; // Warning: Current ignored by frontend. Would be for metadata about the query.
 }
 
+
+//-----------------------------------------------------------------
+// Stream -- EXPERIMENTAL and is subject to change until 8.0
+//-----------------------------------------------------------------
+
+service Stream {
+  // Describe the capibilites for a given stream or an error if invalid
+  rpc GetStreamCapabilities(StreamRequest) returns (stream StreamCapabilities);
```

Comment:
Can't this be combined with `CanSubscribe` rpc? Or it's going to be called in another point of time?